pkg/connector/user.go

```diff
@@ -83,6 +84,34 @@ func (o *userResourceType) List(ctx context.Context, _ *v2.ResourceId, pt *pagin
 			sdkResource.WithUserProfile(userProfile(ctx, user)),
 			sdkResource.WithDetailedStatus(o.userStatus(ctx, user)),
 		}
+
+		if user.ThumbnailPhotoUrl != "" {
+			traitOpts = append(traitOpts, sdkResource.WithUserIcon(&v2.AssetRef{
+				Id: user.ThumbnailPhotoUrl,
+			}))
+		}
+		if user.Archived || user.Suspended {
+			traitOpts = append(traitOpts, sdkResource.WithStatus(v2.UserTrait_Status_STATUS_DISABLED))
+		}
+		if user.IsEnrolledIn2Sv {
+			traitOpts = append(traitOpts, sdkResource.WithMFAStatus(
+				&v2.UserTrait_MFAStatus{MfaEnabled: true},
+			))
+		}
+		if user.DeletionTime != "" {
+			traitOpts = append(traitOpts, sdkResource.WithStatus(v2.UserTrait_Status_STATUS_DELETED))
+		}
+		if user.CreationTime != "" {
+			if t, err := time.Parse("2006-01-02T15:04:05-0700", user.CreationTime); err == nil {
```

Comment:
You should be able to use `time.RFC3339` for this string

Reply:
[The docs](https://developers.google.com/admin-sdk/directory/reference/rest/v1/users) say it's an ISO8601 date, which RFC3339 is a subset of. ISO8601 allows the time zone offset to be "Z" or "+0000", which isn't RFC3339 compliant.

Reply:
whew https://ijmacd.github.io/rfc3339-iso8601/